kafka-persistence/src/main/scala/com/evolutiongaming/kafka/flow/kafkapersistence/KafkaPersistence.scala

```diff
@@ -0,0 +1,167 @@
+package com.evolutiongaming.kafka.flow.kafkapersistence
+
+
+import cats.data.NonEmptyList
+import cats.effect.Sync
+import cats.effect.concurrent.Ref
+import cats.implicits._
+import cats.mtl.MonadState
+import cats.{FlatMap, Monad, data}
+import com.evolutiongaming.catshelper.{BracketThrowable, FromTry, Log}
+import com.evolutiongaming.kafka.flow.KafkaKey
+import com.evolutiongaming.kafka.flow.key.{Keys, KeysOf}
+import com.evolutiongaming.kafka.flow.persistence.{PersistenceOf, SnapshotPersistenceOf}
+import com.evolutiongaming.kafka.flow.snapshot.{SnapshotDatabase, Snapshots, SnapshotsOf}
+import com.evolutiongaming.kafka.journal.util.SkafkaHelper._
+import com.evolutiongaming.skafka.consumer.{ConsumerConfig, ConsumerOf, ConsumerRecord, WithSize}
+import com.evolutiongaming.skafka.producer.{Producer, ProducerRecord}
+import com.evolutiongaming.skafka.{FromBytes, Offset, Partition, ToBytes, Topic, TopicPartition}
+import com.evolutiongaming.sstream.Stream
+import com.olegpy.meow.effects._
+import monocle.macros.GenLens
+import scodec.bits.ByteVector
+
+import scala.concurrent.duration._
+import scala.util.control.NoStackTrace
+
+final case class KafkaPersistence[F[_], K, S, A](
+                                                  keysOf: KeysOf[F, K],
+                                                  snapshots: SnapshotPersistenceOf[F, K, S, A],
+                                                  onRecoveryFinished: F[Unit]
+                                                )
+
+object KafkaPersistence {
+
+  private type Record = ConsumerRecord[String, ByteVector]
```

Comment:
If we are depending on `kafka-journal` stuff anyway, for now, then it might make sense to just use `ConsRecord`, which is defined like following: `type ConsRecord = ConsumerRecord[String, ByteVector]`.

Reply:
Great point, done!